src/object_store.cpp

```diff
@@ -93,6 +93,9 @@ void add_index(Table& table, size_t col)
 
 void insert_column(Group& group, Table& table, Property const& property, size_t col_ndx)
 {
+    if (property.type == PropertyType::LinkingObjects) {
+        throw std::logic_error("Cannot directly insert a LinkingObject column, it must be an artifact of an existing link column.");
```

Comment:
This should just be an assert since it should not be possible to hit it.

The error message has a comma splice and it's "LinkingObjects".